testsuite/integration-arquillian/tests/base/src/test/java/org/keycloak/testsuite/oid4vc/issuance/signing/OID4VCIssuerEndpointTest.java

```diff
@@ -458,7 +486,7 @@ public void testCredentialIssuanceWithAuthZCode() throws Exception {
 
             // 1. Get authoriZation code without scope specified by wallet
             // 2. Using the code to get accesstoken
-            String token = getBearerToken(oauth.openid(false).scope(null));
+            String token = getBearerToken(oauth.openid(false).scope("scope-test-credential"));
```

Comment:
Wouldn´t it be better to create another test to ensure it works with and without the scope? The current test also does not match its description "Get authoriZation code without scope specified by wallet"

Reply:
@wistefan Thank you for your comments. I prepared the following three types of test:
- Authz Code Flow, with scope, scope matching
- Authz Code Flow, with scope, scope un-matching
- Authz-Code Flow, without scope